### PR TITLE
release synthetix:3.0.0-alpha.3

### PR DESCRIPTION
**Pull request by mjlescano**

### CODE REVIEW

File: packages/common-config/hardhat.config.ts [hunk 1]
```diff
@@ -12,8 +12,6 @@ import path from 'node:path';
 
 // Load common .env file from root
 dotenv.config({ path: path.resolve(__dirname, '..', '..', '.env') });
-// Load common .env file at ./packages/config-common/.env
-dotenv.config({ path: path.resolve(__dirname, '.env') });
```
Comment (mjlescano) [inline packages/common-config/hardhat.config.ts:16]: Remove the option to use the `./packages/config-common/.env` file

File: packages/common-config/hardhat.config.ts [hunk 2]
```diff
@@ -73,6 +71,7 @@ const config = {
     apiKey: {
       mainnet: process.env.ETHERSCAN_API_KEY,
       rinkeby: process.env.ETHERSCAN_API_KEY,
+      goerli: process.env.ETHERSCAN_API_KEY,
```
Comment (mjlescano) [inline packages/common-config/hardhat.config.ts:74]: We were missing the goerli config 😬

File: packages/synthetix-main/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@synthetixio/main",
-  "version": "3.0.0-alpha.2",
+  "version": "3.0.0-alpha.3",
```
Comment (mjlescano) [inline packages/synthetix-main/package.json:3]: 🚀

File: packages/common-config/hardhat.config.ts (hunk 1 above)
Comment (dbeal-eth) [reply packages/common-config/hardhat.config.ts:16]: :pray:

File: packages/common-config/hardhat.config.ts (hunk 2 above)
Comment (dbeal-eth) [reply packages/common-config/hardhat.config.ts:74]: not sure why that was missing but :pray:

File: packages/synthetix-main/package.json (hunk 3 above)
Comment (dbeal-eth) [reply packages/synthetix-main/package.json:3]: :ship:

**Review by dbeal-eth — APPROVED** [review]

**Comment by dbeal-eth** [comment]
@mjlescano package-lock.json checks are failing?

**Pull request merged**